src/ShellCheck/Checks/ShellSupport.hs

```diff
@@ -283,7 +283,7 @@ checkBashisms = ForShell [Sh, Dash] $ \t -> do
             ("export", ["-p"]),
             ("printf", []),
             ("read", if isDash then ["r", "p"] else ["r"]),
-            ("ulimit", ["f"])
+            ("ulimit", ["H", "S", "t", "f", "d", "s", "c", "m", "l", "p", "n"])
```

Comment:
Nice! POSIX still only allows `f` though, so you can copy the pattern from the line above to make sure the warnings are still in place for `/bin/sh` (and maybe add one test case to verify that dash `ulimit -c 0` is fine, and one to make sure `sh` still gets a warning)

Reply:
Updated, ptal :)